bcrs-ui/public/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <title>hello-world</title>
+    <link rel="icon" href="dummy-not-used-do-not-remove">
```

Comment:
FYI, this line needs to be here for IE11 to work but does nothing in other browsers.

Reply:
Interesting! Thanks for the info.